pandas/core/groupby/groupby.py

```diff
@@ -515,6 +515,15 @@ def indices(self) -> dict[Hashable, npt.NDArray[np.intp]]:
         """
         Dict {group name -> group indices}.
 
+        See Also
+        --------
+        core.groupby.DataFrameGroupBy.indices : Provides a mapping of group names to
+         positions of the elements in object.
```

Comment:
Indent continuations by 4 spaces rather than one.

Reply:
`group rows` instead of `elements` I think. And it seems to me you can remove `in object`.

Reply:
Thank you for reviewing!